src/components/tour-table.tsx

```diff
@@ -7,21 +7,20 @@ import mapStoreToProps from '../redux/mapStoreToProps';
 import TableTheme from '../lib/tourTableTheme';
 
 type TourTableProps = {
-  dispatch: (...args: any[]) => any;
-  tourUpdated: boolean;
-  tour: any[];
-  auth: {token: string};
+  dispatch?: (...args: any[]) => any;
+  tourUpdated?: boolean;
+  tour?: Array<number[] | string[]>;
```

Comment:
@Unesdala this should be an array of Tour where Tour is an interface that contains the attributes of a single table row or document object in this case since we are using MongoDB. Lets skip this for now, I think i'm going to change the naming of this data so that it is easier to understand whereas Songs is an array of Song, I think i'll change this to be Concert and Concerts is an array of Concert.